projects/kenshi/index.js

```diff
@@ -0,0 +1,18 @@
+const { sumTokensExport } = require("../helper/unwrapLPs");
+
+const contracts = [
+  "0x4FDbc784f09BEAcEFCBDcBD7c724015416edcE53", // Protocol & Incentives
+  "0xD59321c8266534dac369F0eFABDD5b815F1a5eb6", // Treasury
+  "0xE894BD5Ec531EC8AAe856AFC3E0Fc948Ab22Efb4", // Staking
```

Comment:
we can count the kenshi tokens here under staking, value in rest of the addresses can be counted towards treasury metric.